public/static/favicons/site.webmanifest

```diff
@@ -1,14 +1,72 @@
 {
-  "name": "",
-  "short_name": "",
+  "name": "TailwindBlog",
+  "short_name": "TailwindBlog",
+	"display": "standalone",
+  "description": "A blog created with Next.js and Tailwind.css",
+  "lang": "en",
+	"theme_color": "#ffffff",
+  "background_color": "#ffffff",
+  "start_url": "/",
   "icons": [
     {
-      "src": "/android-chrome-96x96.png",
-      "sizes": "96x96",
+      "src": "/static/favicons/android-chrome-192x192.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    },
+    {
+      "src": "/static/favicons/android-chrome-512x512.png",
+      "sizes": "512x512",
       "type": "image/png"
     }
   ],
-  "theme_color": "#000000",
-  "background_color": "#000000",
-  "display": "standalone"
+  "shortcuts": [
+    {
+      "name": "Open Home",
+      "short_name": "Home",
+      "url": "/",
+      "description": "Home page",
+      "icons": [
+        {
+          "src": "/static/favicons/android-chrome-192x192.png",
+          "sizes": "192x192"
+        }
+      ]
+    },
```

Comment:
I would leave the `Open Home` shortcut away, because the link to the home page is automatically added and Chrome on mobile can only display 3 shortcuts.

Reply:
Thanks for your suggestion, I will change that on the next commit.